gitpod-io/gitpod
# [JetBrains] Make new features available in stable version

## Description
Moving ready features from `latest` to `stable`

## Related Issue(s)

Fixes #12156

## How to test
1. Launch a workspace in [prev env](https://andreafalz1c05f7f767.preview.gitpod-dev.com) using JetBrains stable
2. Open a repository that has multiple tasks defined (like gitpod repo)
3. Make sure the following features are available:
    - CPU/Memory visible in Backend Control Center
    - Confirm if a terminal widget appears for each task defined on .gitpod.yml


## Release Notes

```release-note
JetBrains: Display Workspace CPU/Memory usage in Backend Control Center.
JetBrains: Workspaces created from repositories with tasks defined on .gitpod.yml will start with one terminal opened for each task.
```

## Documentation



## Werft options:
